sklearn/ensemble/voting_classifier.py

```diff
@@ -54,6 +54,12 @@ class VotingClassifier(_BaseComposition, ClassifierMixin, TransformerMixin):
         the sums of the predicted probabilities, which is recommended for
         an ensemble of well-calibrated classifiers.
 
+    verbose: int, optional
+        The verbosity level: if non zero, progress messages are printed.
+        Above 50, the output is sent to stdout. The frequency of the messages
+        increases with the verbosity level. If it more than 10, all iterations
```

Comment:
"iterations" does not really apply here.

How many estimators does one need to have in a VotingClassifier for anything lower than 10 to be helpful here?

Reply:
Good point! Thanks a lot for your review.

I replace "iterations" by "`fit` calls" in fcbfbcd.

I would say that a value lower than 10 is useful when:
- the number of estimator is big (let say 500)
- each estimator is pretty quick to fit (let say 1 second)
- n_jobs is strictly greater than 1

I'm thinking of a situation similar to this call:
```python
from time import sleep
from sklearn.externals.joblib import Parallel, delayed
r = Parallel(n_jobs=2, verbose=5)(delayed(sleep)(1.) for _ in range(500))
```

In this case, the user may want to have feedback to know that (s)he has 4.16 minutes to get a coffee (`500 / 2 * 1 / 60`), but without being spammed by 500 lines. Does it make sense?